src/components/navigation/header/drawer/userInfo/SelectedTenantInfo.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC } from 'react';
+import { ManagedSeller } from '@smg-automotive/auth';
+
+import Stack from 'src/components/stack';
+import { GarageIcon } from 'src/components/icons';
+import Box from 'src/components/box';
+
+type Props = {
+  selectedTenant: ManagedSeller;
+};
+
+const SelectedTenantInfo: FC<Props> = ({ selectedTenant }) => (
+  <Stack direction="row">
```

Comment:
are we using Stack inside the header already? browser support for flex gap is below what we define in our browserslist

Reply:
Yes, it's already used in the drawer